src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -135,7 +135,7 @@
 	public String aspectRatioVideoTrack;
 	private int videoBitDepth = 8;
 
-	private byte thumb[];
+	private volatile byte[] thumb = null;
```

Comment:
@atamariya These objects are used by multiple threads. Volatile isn't enough, a proper lock should be in place but since it's an array (fixed size) and it's very unlikely that individual bytes will be modified in an image, ```volatile``` is a "quick and dirty" way to make it a bit more protected. It makes sure that the different threads will at least reference the same array.